onesignal-settings.php

```diff
@@ -138,41 +138,32 @@ public static function get_onesignal_settings() {
       }
     }
 
-    /*
```

Comment:
I removed this first comment block as I'm not sure if it's still meaningful.  Are these supposed to be caller instructions?